### PR TITLE
Update pdf icon

### PR DESCRIPTION
**Pull request by marcoambrosini**
| Before | After |
| - | - | 
| Screenshot 2024-05-07 at 13 04 28 | Screenshot 2024-05-13 at 10 55 01 |

### CODE REVIEW

File: core/img/filetypes/application-pdf.svg [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-  <path d="M0 0h24v24H0V0z" fill="none" />
-  <path fill="#dc5047"
-    d="M20 2H8c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-8.5 7.5c0 .83-.67 1.5-1.5 1.5H9v1.25c0 .41-.34.75-.75.75s-.75-.34-.75-.75V8c0-.55.45-1 1-1H10c.83 0 1.5.67 1.5 1.5v1zm5 2c0 .83-.67 1.5-1.5 1.5h-2c-.28 0-.5-.22-.5-.5v-5c0-.28.22-.5.5-.5h2c.83 0 1.5.67 1.5 1.5v3zm4-3.75c0 .41-.34.75-.75.75H19v1h.75c.41 0 .75.34.75.75s-.34.75-.75.75H19v1.25c0 .41-.34.75-.75.75s-.75-.34-.75-.75V8c0-.55.45-1 1-1h1.25c.41 0 .75.34.75.75zM9 9.5h1v-1H9v1zM3 6c-.55 0-1 .45-1 1v13c0 1.1.9 2 2 2h13c.55 0 1-.45 1-1s-.45-1-1-1H5c-.55 0-1-.45-1-1V7c0-.55-.45-1-1-1zm11 5.5h1v-3h-1v3z" />
+<svg width="96" height="96" viewBox="0 0 96 96" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M76 12H20C15.6 12 12 15.6 12 20V76C12 80.4 15.6 84 20 84H76C80.4 84 84 80.4 84 76V20C84 15.6 80.4 12 76 12ZM38 46C38 49.2 35.2 52 32 52H28V60H22V36H32C35.2 36 38 38.8 38 42V46ZM58 54C58 57.2 55.2 60 52 60H42V36H52C55.2 36 58 38.8 58 42V54ZM74 42H68V46H74V52H68V60H62V36H74V42ZM48 42H52V54H48V42ZM28 42H32V46H28V42Z" fill="#FF2116"/>
```
Comment (susnux) [inline core/img/filetypes/application-pdf.svg:2]: Should we not use the MDI icon?

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 3H5C3.9 3 3 3.9 3 5V19C3 20.1 3.9 21 5 21H19C20.1 21 21 20.1 21 19V5C21 3.9 20.1 3 19 3M9.5 11.5C9.5 12.3 8.8 13 8 13H7V15H5.5V9H8C8.8 9 9.5 9.7 9.5 10.5V11.5M14.5 13.5C14.5 14.3 13.8 15 13 15H10.5V9H13C13.8 9 14.5 9.7 14.5 10.5V13.5M18.5 10.5H17V11.5H18.5V13H17V15H15.5V9H18.5V10.5M12 10.5H13V13.5H12V10.5M7 10.5H8V11.5H7V10.5Z" fill="#FF2116" /></svg>
```

**Review by jancborchardt — DISMISSED** [review]
As discussed with @marcoambrosini, we should keep the previous coloring of the icon because it looks less intense and much better together with the default folder color.

File: core/img/filetypes/application-pdf.svg (hunk 1 above)
Comment (marcoambrosini) [reply core/img/filetypes/application-pdf.svg:2]: You mean dropping the svg file and using the one from the library?

**Review by ShGKme — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**